requirements-dev.txt

```
@@ -5,3 +5,4 @@ mock>=2.0.0
 jinja2>=2.8
 pylint>=1.5.5
 urllib3 >= 1.15
+isi_sdk_7_2>=0.1.2
```

Consider referencing `requirements.txt` instead of duplicating it here.
```suggestion
-r requirements.txt
```